torch/_export/serde/serialize.py

```diff
@@ -762,12 +777,14 @@ def serialize_graph(self, graph_module: torch.fx.GraphModule) -> Graph:
 
     def serialize(self, graph_module: torch.fx.GraphModule) -> GraphModule:
         graph = self.serialize_graph(graph_module)
+        serialized_constants = base64.b64encode(serialize_torch_artifact(self.constants)).decode('utf-8')
```

Comment:
base64 sounds like a json specific thing. I think maybe we wanna put it into the jsonencoder layer?